public/data/AliReza1083.json

```
@@ -11,9 +11,14 @@
     },
     {
       "name": "Follow me on Twitter",
-      "url": "https://twitter.com/webDeveloper83",
+      "url": "https://twitter.com/Ali_Developer05",
       "icon": "twitter"
     },
+    {
+      "name": "Check out my Portfolio",
+      "url": "https://gleaming-marshmallow-fb9096.netlify.app/",
+      "icon": "global"
```

Not sure there's a `global` icon.
```suggestion
      "icon": "globe"
```